.github/workflows/add_assignee_when_pr_opened.yml

```diff
@@ -0,0 +1,48 @@
+# See https://confluence.diamond.ac.uk/x/tJqQC for instructions if this needs a new token
+name: Assign author of pull request to referenced issue that was not assigned to issue ticket beforehand
+on:   
+ pull_request:
+      types: [ready_for_review, opened, review_requested]
+jobs:
+  assign_pr_author_as_assignee:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get project data
+        # This will get information about the project in general and add it to the environment.
+        # * ISSUE_REFERENCED_ID - The ID for the Issue that was referenes at Fixes#999
+        # * AUTHOR_ID - The ID of the person who made a pull request so that the author can be assigned to the Fixes#999
```

Comment:
Should: This comment is wrong, it doesn't output `ISSUE_REFERENCED_ID` but instead `ASSIGNABLE_ID`